src/api/Snapshot.ts

```diff
@@ -341,6 +378,102 @@ export class Snapshot extends API {
     const vp = await this.getLatestScores(space, [address])
     return Object.values(vp)[0]
   }
+
+  async getVotes(space: string, start: Date, end: Date) {
```

Comment:
Also, I think we shouldn't need the space as an argument, right? The Snapshot API class already knows the space based on the env, and we aren't querying more than one space per environment.

It could have it inside the method:

```typescript
 variables: {
              space: SNAPSHOT_SPACE,
              start: getQueryTimestamp(start.getTime()),
              end: getQueryTimestamp(end.getTime()),
              first,
              skip,
            },
            ```